keys/constants.go

```diff
@@ -117,6 +117,10 @@ var (
 	// LocalMax is the end of the local key range.
 	LocalMax = LocalPrefix.PrefixEnd()
 
+	// TableDataPrefix prefixes all Table data to aid in transitioning
+	// key:value data to Table data, and for ease of debugging.
+	TableDataPrefix = proto.Key("table-")
```

Comment:
I'd place this key definition down lower in the list of constants. It feels out of place here wedged between local keys and system keys. Down at the very bottom of this `var` block seems right.